src/main/java/br/com/homefood/entity/Receita.java

```diff
@@ -1,15 +1,30 @@
 package br.com.homefood.entity;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EnumType;
+import javax.persistence.Enumerated;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
 import br.com.homefood.classenum.TipoReceita;
 
+@Entity
+@Table(name = "Receita")
```

Comment:
Mesma coisa sobre o nome da tabela, em letra minúscula
```suggestion
@Table(name = "receita")
```